inc/field/dropdownfield.class.php

```diff
@@ -715,6 +715,7 @@ public function parseObjectProperties(
       // We need english locale to search searchOptions by name
       $oldLocale = $TRANSLATE->getLocale();
       $TRANSLATE->setLocale("en_GB");
+      $_SESSION['glpilanguage'] = "en_GB";
```

Comment:
We could maybe replace ```$TRANSLATE->setLocale("en_GB");``` and  ``` $_SESSION['glpilanguage'] = "en_GB";``` wih a single call to Session::loadLanguage().

Reply:
if I'm not mistaken ```Session::loadLanguage().``` does not overload ```$_SESSION['glpilanguage']``` with ```$forcelang```

Reply:
Indeed. Then I approve this solution.